### PR TITLE
1.1.1 Hotfix

### LINKED ISSUE #219: Popup is behind a card on macro overview
**Describe the bug**

**To Reproduce**
1. Go to 'Macro overview'
2. Click on 'Three dot hamburger menu'
4. See error - the menu is behind the card if there is one on the right side

**Screenshots / Video**
telegram-cloud-photo-size-4-6021336277326086829-m

**Desktop (please complete the following information):**
 - OS: Win 11

**Wootomation version**
1.1.0

### LINKED ISSUE #225: Right modifiers don't work
**Describe the bug**
Right side modifiers dont work
https://discord.com/channels/167181566978555904/1063455997575843921/1228768620381995009

**To Reproduce**
1. Set the right side trigger keys

**Screenshots / Video**
image

**Desktop (please complete the following information):**
 - OS: Win 10

**Wootomation version**
1.1.0

### PR DESCRIPTION
**Pull request by medzernik**
Fixes #219 
Fixes #225

### CODE REVIEW

File: eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import js from "@eslint/js";
+
+export default [
+  js.configs.recommended,
+  {
+    rules: {
+      "no-unused-vars": "warn"
+    }
+  }
+];
```
Comment (simon-wh) [inline eslint.config.js]: This eslint configuration doesn't seem to be equivalent to the previous?

Comment (medzernik) [reply eslint.config.js]: hmm true.. tho i haven't encountered eslint working properly in my IDE so figured i shouldn't spend too much time on it.. tho i can look it over a bit more in detail

Comment (simon-wh) [reply eslint.config.js]: I would try and make sure the rules are all the same, the old config had a fair amount of customisation

Comment (medzernik) [reply eslint.config.js]: Didn't update eslint in the end

**Pull request closed without merging**